src/content/docs/apis/nerdgraph/examples/nerdgraph-nrql-tutorial.mdx

```diff
@@ -148,36 +151,41 @@ When using NerdGraph to explore your data, you can use the `suggestedFacets` fie
   >
     Here's an example of returning suggested attributes for faceting [transaction](/docs/insights/insights-data-sources/default-data/apm-default-event-attributes-insights#transaction-event) counts. The response suggests the `host` attribute. Faceting by `host` can reveal that one host is servicing more requests than other hosts.
 
-    ```graphql
+    ```
```

Comment:
@brnhensley should this have the json code fence? This looks like a response?

Reply:
this one is graphql, but I did not leave the identifier because it has inline HTML: 
```html
"<a href="/docs/insights/nrql-new-relic-query-language/nrql-resources/nrql-syntax-components-functions#state-select">SELECT</a> count(*) from <a href="https://docs.newrelic.com/docs/insights/insights-data-sources/default-data/apm-default-event-attributes-insights#transaction-event">Transaction</a> <a href="/docs/insights/nrql-new-relic-query-language/nrql-resources/nrql-syntax-components-functions#sel-timeseries">TIMESERIES</a>"
```

which will appear as plain text if you try to use an identifier to it, instead of like:

```js
"SELECT count(*) from Transaction TIMESERIES"
```

I did adjust the indentation though.